components/grid.css

```diff
@@ -0,0 +1,27 @@
+.grid {
+	display: grid;
+}
+
+.grid-2 {

```

Comment:
Probably rename it in a more semantic way

Reply:
done